docs/index.adoc

```diff
@@ -61,8 +61,13 @@ How it works: annotate your class and picocli initializes it from the command li
 
 TIP: Picocli also provides a <<Programmatic API,programmatic API>>, separately from the annotations API.
 
-=== Example application
-The example below shows a short but fully functional picocli-based `checksum` command line application:
+=== Example application: Checksum
```

Comment:
I am not in favor of adding the word ": Checksum"
* it will cause the title to wrap in the TOC
* it distracts from the purpose: the purpose is to provide an example that gives a taste of picocli's power and ease of use. The fact that it is also a useful program by itself is secondary.

Reply:
I saw the title wrap in the TOC already and didn't like it, too.
Reverted to original version.